lib/matplotlib/projections/polar.py

```diff
@@ -165,7 +165,7 @@ def transform_non_affine(self, xy):
         # docstring inherited
         x, y = xy.T
         r = np.hypot(x, y)
-        theta = (np.arctan2(y, x) + 2 * np.pi) % (2 * np.pi)
```

Comment:
This is not equivalent for negative numbers.

Reply:
I think this is the same:

```
In [3]: (-1 ) % (2 * np.pi)
Out[3]: 5.283185307179586

In [4]: (-1 + 2 * np.pi) % (2 * np.pi)
Out[4]: 5.283185307179586

```